### PR TITLE
PA-1684: Fix existing locked bucket cases

### PR DESCRIPTION
**Pull request by kshithijiyer-px**
**What this PR does / why we need it**:
- [x] Fix existing locked bucket cases

Individual testcase runs:
https://aetos.pwx.purestorage.com/resultSet/testSetID/360397
https://aetos.pwx.purestorage.com/resultSet/testSetID/359217
https://aetos.pwx.purestorage.com/resultSet/testSetID/359640 

Full run:
https://aetos.pwx.purestorage.com/resultSet/testSetID/364072

**Which issue(s) this PR fixes** (optional)
Closes # PA-1684

**Special notes for your reviewer**:

### CODE REVIEW

**Review by kphalgun-px — APPROVED** [review]
LGTM

File: tests/backup/backup_locked_bucket_test.go [hunk 1]
```diff
@@ -104,16 +105,15 @@ var _ = Describe("{BackupAlternatingBetweenLockedAndUnlockedBuckets}", func() {
 			modes := [2]string{"GOVERNANCE", "COMPLIANCE"}
 			for _, provider := range providers {
 				for _, mode := range modes {
-					CredName := fmt.Sprintf("%s-%s-%v", "cred", provider, time.Now().Unix())
-					bucketName := fmt.Sprintf("%s-%s-%s-locked", provider, getGlobalLockedBucketName(provider), strings.ToLower(mode))
-					backupLocation = fmt.Sprintf("%s-%s-%s-lock", provider, getGlobalLockedBucketName(provider), strings.ToLower(mode))
+					bucketName := fmt.Sprintf("%s-%s-%v", getGlobalLockedBucketName(provider), strings.ToLower(mode), time.Now().Unix())
+					backupLocation = fmt.Sprintf("%s-%s-lock", getGlobalLockedBucketName(provider), strings.ToLower(mode))
```
Comment (kphalgun-px) [inline tests/backup/backup_locked_bucket_test.go:109]: How about including time stamp?

Comment (kshithijiyer-px) [reply tests/backup/backup_locked_bucket_test.go:109]: It'll have run hash passed to it through the env hence we won't need it.

File: tests/backup/backup_locked_bucket_test.go
Comment (kphalgun-px) [inline tests/backup/backup_locked_bucket_test.go:1]: log.InfoD is missing right after some steps.

Comment (kshithijiyer-px) [reply tests/backup/backup_locked_bucket_test.go:1]: Okay let me push it with the other fix

File: tests/backup/backup_locked_bucket_test.go [hunk 2]
```diff
@@ -21,7 +21,8 @@ import (
 // This testcase verifies alternating backups between locked and unlocked bucket
 var _ = Describe("{BackupAlternatingBetweenLockedAndUnlockedBuckets}", func() {
 	var (
-		appList = Inst().AppList
+		appList  = Inst().AppList
```
Comment (mkoppal-px) [inline tests/backup/backup_locked_bucket_test.go:24]: nit: There is a double space

File: tests/backup/backup_locked_bucket_test.go [hunk 3]
```diff
@@ -840,7 +853,7 @@ var _ = Describe("{DeleteLockedBucketUserObjectsFromAdmin}", func() {
 				createRestore := func(backupName string, restoreName string, namespace string) {
 					defer GinkgoRecover()
 					defer wg.Done()
-					customNamespace := fmt.Sprintf("custom-%s-%v", namespace, time.Now().Unix())
+					customNamespace := fmt.Sprintf("custom-%s", namespace)
```
Comment (mkoppal-px) [inline tests/backup/backup_locked_bucket_test.go:856]: Why did we remove the timestamp?

File: tests/backup/backup_locked_bucket_test.go (hunk 2 above)
Comment (kshithijiyer-px) [reply tests/backup/backup_locked_bucket_test.go]: Done

File: tests/backup/backup_locked_bucket_test.go (hunk 3 above)
Comment (kshithijiyer-px) [reply tests/backup/backup_locked_bucket_test.go:856]: The namespace name becomes more than 64 character and causes the case to fail with: 
```


rpc error: code = Unknown desc = failed to create backup location [autogenerated-backup-location-1696056535] CR: rpc error: code = NotFound desc = failed to create backup location CR [autogenerated-backup-location-1696056535]: namespaces "custom-postgres-backup-pxbackuptask-0-09-30-06h47m47s-1696056661" not found
--


<br class="Apple-interchange-newline">
```

**Review by mkoppal-px — APPROVED** [review]
LGTM

**Pull request merged**